src/serlio/utils/MELScriptBuilder.h

```diff
@@ -0,0 +1,77 @@
+/**
+ * Serlio - Esri CityEngine Plugin for Autodesk Maya
+ *
+ * See https://github.com/esri/serlio for build and usage instructions.
+ *
+ * Copyright (c) 2012-2019 Esri R&D Center Zurich
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "utils/MayaUtilities.h"
+
+#include "maya/MGlobal.h"
+
+#include <array>
+#include <cassert>
+#include <sstream>
+#include <string>
+
+class MaterialColor;
+
+class MELVariable : public mu::NamedType<std::wstring, struct MelVariableName> {

```

Comment:
is there a reason, why we don't use the class name `MELVariable` directly as the tag?

Reply:
done